guides/common/modules/proc_using-the-host-details-tab.adoc

```diff
@@ -1,5 +1,5 @@
 [id="using-the-host-details-tab_{context}"]
-= Using the Host Details Tab
+= Using the Details tab
```

Comment:
unsure about this change. Either way, please ensure the anchor and file name matches your title.

Reply:
Done